docs/develop/01_sdk/02_Cookbook/01_Manage a KILT DID/04_full_did_batch.md

```diff
@@ -0,0 +1,18 @@
+---
+id: full-did-batch
+title: Sign extrinsics with a full DID
+---
+
+import CodeBlock from '@theme/CodeBlock';
+import Example9 from '!!raw-loader!@site/code-examples/core_features/did/9_did.ts';
+
+Full DIDs can also be used to batch multiple extrinsics that require the signature of the DID subject.
+For instance, a batch could include the creation of two different CTypes in the same operation.
+This would save the user one block time and one signature, as multiple extrinsics are batched and signed once, and they are submitted and executed in the same block.
```

Comment:
W: 
```suggestion
This would save the user time of one block and one signature, as multiple extrinsics are batched and signed at once, and they are submitted and executed in the same block.
```

Reply:
https://github.com/KILTprotocol/docs/pull/101/commits/1bba37b958dc8d3b95ebb1a41db2825c6fb2bb06